configure.ac

```
@@ -0,0 +1,305 @@
+AC_PREREQ([2.59])
+AC_INIT([conserver],
+  [m4_esyscmd([build-aux/git-version-gen .tarball-version])],
+  [bryan@conserver.com], [],
+  [https://www.conserver.com/])
+configure_flags="$*"
+AC_CONFIG_SRCDIR([conserver/main.c])
+AC_CONFIG_AUX_DIR([build-aux])
+AC_USE_SYSTEM_EXTENSIONS
+AM_INIT_AUTOMAKE([
+  1.7.9
+  dist-xz
+  foreign
+  no-dist-gzip
+  subdir-objects
+  tar-pax
+])
+
+# Basic autotools configuration.
+m4_ifdef([AM_SILENT_RULES],
+  [AM_SILENT_RULES([yes])],
+  [AC_SUBST([AM_DEFAULT_VERBOSITY], [1])])
+
+AC_CANONICAL_HOST
+AC_C_CONST
+AC_PROG_CC_C99
+AC_PROG_INSTALL
+
+AC_DEFINE_UNQUOTED([VERSION_TEXT], ["conserver.com version"], [Version banner])
+version_int=`echo $PACKAGE_VERSION | awk -F. '{print $1 * 1000000 + $2 * 1000 + $3}'`
+AC_DEFINE_UNQUOTED([VERSION_UINT], [$version_int],
+  [Conserver version as unsigned int])
+
+AC_DEFINE_UNQUOTED([COPYRIGHT],
+  ["Copyright 1990 The Ohio State University.\n\
+Copyright 1992 Purdue Research Foundation.\n\
+Copyright 1998 GNAC, Inc.\n\
+Copyright 2000 conserver.com.\n\
+All rights reserved.\n"],
+  [Conserver copyright])
+
+usrsbin_execdir='${exec_prefix}/sbin'
+AC_SUBST([usrsbin_execdir])
+
+# Check pkg-config availability.
+m4_ifndef([PKG_PROG_PKG_CONFIG],
+  [m4_fatal(
+[Could not locate the pkg-config autoconf macros.  These are usually located
+in /usr/share/aclocal/pkg.m4.  If your macros are in a different location,
+try setting the environment variable ACLOCAL_OPTS="-I/other/macro/dir"
+before running ./autogen.sh again.])
+])
+PKG_PROG_PKG_CONFIG
+
+# Check headers.
+AC_HEADER_STDC
+AC_HEADER_SYS_WAIT
+AC_HEADER_TIME
+
+AC_CHECK_HEADERS([ \
+  bsm/audit.h \
+  hpsecurity.h \
+  libutil.h \
+  prot.h \
+  pty.h \
+  sys/ioctl.h \
+  sys/proc.h \
+  util.h \
+])
+
+# Check functions.
+AC_CHECK_FUNCS([ \
+  bigcrypt \
+  getaudit \
+  getaudit_addr \
+  getdtablesize \
+  getlogin \
+  getopt \
+  getprpwnam \
+  getrlimit \
+  getsid \
+  getspnam \
+  gettimeofday \
+  getuserattr \
+  getutent \
+  grantpt \
+  inet_aton \
+  iscomsec \
+  memcmp \
+  memcpy \
+  memmove \
+  memset \
+  ptsname \
+  putenv \
+  setgroups \
+  setlinebuf \
+  setpgrp \
+  setsid \
+  setsockopt \
+  setttyent \
+  setvbuf \
+  sigaction \
+  strerror \
+  strlcpy \
+  sysconf \
+  tcgetattr \
+  tcgetpgrp \
+  tcsendbreak \
+  tcsetattr \
+  tcsetpgrp \
+  unlockpt \
+])
+
+AC_SEARCH_LIBS([openpty], [util], [UTIL_LIBS=""], [UTIL_LIBS="-lutil"])
```

You got this backwards! This should be:

```suggestion
AC_SEARCH_LIBS([openpty], [util], [UTIL_LIBS="-lutil"], [UTIL_LIBS=""])
```